release-notes/3.0/preview/api-diff/preview9/README.md

```diff
@@ -0,0 +1,7 @@
+# .NET Core 3.0 Preview 9 API Changes
+
+The following API changes were made in .NET Core 3.0 Preview 8:
+
+- [.NET Core](./.Net/3.0-preview9.md)
+- [.NET Core NuGet Packages](./.Net/standalone-packages/3.0-preview9-standalone-packages.md)
+- [ASP.NET Core](./ASP.Net/3.0-preview9.md)
```

Comment:
nit: Lower-case ASP.Net to match the filename: `Asp.Net/3.0-preview9.md`

```suggestion
- [ASP.NET Core](./Asp.Net/3.0-preview9.md)
```

OR change the file name to be capital and match preview 8. I don't have a strong preference.